unfoldingWord/translationCore
# Verse Edit

#### This pull request addresses:

This pull request implements the verse edit feature. 


#### How to test this pull request:

Set the verse check branch to the branch of the same name. Use the edit feature in verse check, and look at the files that match as well.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1062)
